cfme/tests/containers/test_static_custom_attributes.py

```diff
@@ -14,14 +21,43 @@
     pytest.mark.tier(2)]
 pytest_generate_tests = testgen.generate([ContainersProvider], scope='function')
 
+
+def get_random_string(length):
```

Comment:
well, didn't know about fauxfactory - I'll check this and probably use it.
and yes - there are some printable characters that I want to prevent from using them.
Let me investigate fauxfactory, if it'll yield what I want I'll change accordingly :)